tcmu-runner.service

```diff
@@ -4,5 +4,5 @@ Description=LIO Userspace-passthrough daemon
 [Service]
 Type=dbus
 BusName=org.kernel.TCMUService1
+KillMode=process
```

Comment:
Right now runner restart on failure does not work for any useful case.

Restart on failure only works if IO is not running, which is probably the least useful and uncommon case runner will fail, so for now I am ok with just removing it from systemd. To implement restart support we need to do:

1. if cmd_time_out > 0, fix target_core_user so that commands that have been timeout by the module have their ring buffer resources like data/cmd buffer memory cleaned up when it is safe. So I think when tcmu-runner restarts it needs to figure out which commands have been failed by the module due to timeout a do this cleanup operation on them.

2. If cmd_timeout == 0, some handlers may need to figure out the status of outstanding commands, and we may need to clean them up on the handler side or resetup the runner state for them, and then we need to clean up the kernel resources like in #1 if we are aborting them.